### PR TITLE
Adding inactive-state FastLane Guide text

### PR DESCRIPTION
**Pull request by kategarklavs**
Per issue #725, adding inactive-state copy re: the FastLane guide.

Fixes issue(s) #725 

[CircleCI](https://circleci.com/gh/18F/nsf-sbir/tree/kategarklavs-patch-4)

[:sunglasses: PREVIEW](https://federalist.fr.cloud.gov/preview/18f/nsf-sbir/kategarklavs-patch-4/)

[Preview README for this branch](https://github.com/18F/nsf-sbir/blob/kategarklavs-patch-4/README.md)

Changes proposed in this pull request:
- Adding inactive-state text for the FastLane guide 
-
-

/cc @kmonterr

### CODE REVIEW

File: _timeline/step-6.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: Start your application
 becomes_inactive: true
 description: Once you've reviewed the solicitation, start preparing your Phase I application in FastLane.
-inactive_description: We're not currently accepting proposal applications. Our next solicitation will be released in {{ site.solicitation_released }}.
+inactive_description: Though we're not currently accepting applications, we encourage you to familiarize yourself with our [FastLane guide]({{ site.baseurl }}/fastlane/). This way, once the next solicitation opens, you'll be ready to craft your application. We'll release our next solicitation in {{ site.solicitation_released }}.
```
Comment (line47) [inline _timeline/step-6.md]: the link to `/fastlane/` should be `/fastlane/fastlane-1/`

Comment (kategarklavs) [inline _timeline/step-6.md]: Word! Apologies for the confusion - I'll update this now.

Comment (kategarklavs) [inline _timeline/step-6.md]: Updated! Just merged the PR

**Comment by kategarklavs** [comment]
@line47 OK - I updated the links. Ready for re-review!

**Pull request merged**